code/game/objects/structures/crates_lockers/closets/secure/freezer.dm

```diff
@@ -4,13 +4,11 @@
 	door_anim_squish = 0.22
 	door_anim_angle = 123
 	door_anim_time = 4
+	material_drop = /obj/item/stack/sheet/iron
+	material_drop_amount = 2
 	/// If FALSE, we will protect the first person in the freezer from an explosion / nuclear blast.
 	var/jones = FALSE
 
-/obj/structure/closet/secure_closet/freezer/Initialize(mapload)
-	. = ..()
-	register_context()
-
```

Comment:
why were the contextual tips removed?

Reply:
Because I didn't realize this in the previous PR. but they all are inherited from structure/closet. There are already examines telling you that's it's welded together so it became redundant